src/app/view/components/gameModeButtons.js

```diff
@@ -0,0 +1,49 @@
+import elementCreator from '../../utils/elementCreator';
+
```

Comment:
I think this file is not necessary in this PR ?

Reply:
deleted